libuuu/globals.cpp

```diff
@@ -0,0 +1,3 @@
+#include "libuuu.h"
+int g_verbose = 1;
+bmap_mode g_bmap_mode = bmap_mode::Default;
```

Comment:
Strange, there are many global in libuuu, why only these two varible need a seperate files.

Reply:
These two particular globals are declared with `extern` in `libuuu.h` and `bmap.h` respectively. To compile sources in `libuuu` folder into shared binary you need to define them, so that the binary can be used with `dlopen` etc. For uuu tool you define them in `uuu.cpp` but we don't use that file. 

The `globals.cpp` is used only with the wrapper, and no changes were made to the uuu.

Reply:
bmap implment have some problem.  Let me fix this problem.

Reply:
Please pull latest change,  globals.cpp will not needn't anymore.